data/campaigns/Under_the_Burning_Suns/scenarios/06b_In_the_Domain_of_Dwarves.cfg

```diff
@@ -1123,6 +1131,14 @@
                     time=1000
                 [/delay]
 
+                [sound]
+                    name=thunderstick-miss.ogg
+                [/sound]
+
+                [delay]
+                    time=1800
```

Comment:
Hmm that's a looong delay.

Reply:
Have you tested this in the scenario? I actually fine tuned that value.

Reply:
Nope, but I've made a gazillion cutscenes before. Longest delay I ever used was 1500 and that was already for a crazy long pause. I usually don't go above 1000 personally. Players aren't that patient and longer delays make them think that the game is frozen lol.

Reply:
... There is something going on in that 1800. The duration is that length to accommodate the duration of the sound being played. It is not an empty 1800 pause. Does that make sense? It sounds good when you try it, I actually tried 1000, then 1500, then finally 1800, which was perfect.

Reply:
If it makes any difference, the thunderstick mis-fire sound effect is 1400 ms.